app/views/books/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1><%= @book.title %></h1>
+<p>
+  <%= @book.body %>
```

Comment:
The intention was to post a review's body rather than the body of the book - that would be a lot of content! 
I see you've done this below.